src/Homework2/task1.py

```diff
@@ -0,0 +1,21 @@
+﻿'''
+1. Напишите программу, которая считает общую цену.
+Вводится M рублей и N копеек цена, а также количество S товара
+Посчитайте общую цену в рублях и копейках за L товаров.
+Пример:
+Input: Цена одной вещи 3 рубля 20 копеек, посчитать 3 предмета.
+Output: Общая цена 9 рублей 60 копеек
+Задачу поместите в файл task1.py в папке src/homework2.
+'''
+while True:
```

Comment:
why do we need while here?

Reply:
To check working program with some different input data without restart program